libmscore/cmd.cpp

```diff
@@ -243,9 +243,9 @@ void Score::update()
             if (is.noteEntryMode() && is.segment()) {
                   setPlayPos(is.segment()->tick());
                   }
-            if (_playlistDirty) {
+            if (playlistDirty()) {
```

Comment:
This change (with the `setPlaylictClean()` line below) needs further checking: it makes `playlistChanged` signal be emitted only for master score but not for excerpt scores (as `Score::update()` is usually called in a loop over the master score and all excerpt scores). This may potentially break the code that uses this signal so it is probably worth checking whether this change has any unexpected effects.

Reply:
Well, it turns out that the only thing that could suffer from this potential issue is a pianoroll editor, and its handler for `playlistChanged` signal is just empty... that is strange.
https://github.com/musescore/MuseScore/blob/4899a7cd69cd6594351e10114ef039d6bb79707a/mscore/pianoroll.cpp#L647-L649
Anyway, maybe we can then safely move the very signal to MasterScore?